tgui/packages/tgui/interfaces/PaperSheet/index.tsx

```diff
@@ -304,6 +308,18 @@ const PaperSheetEdit: React.FC<PaperSheetEditProps> = ({
     }
   };
 
+  const onKeyDownHandler = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.key === 'Enter') {
```

Comment:
tgui-core has constants for keys. https://github.com/tgstation/tgui-core/blob/main/lib/common/keys.ts